### DISCUSSION TITLE
add armsom-sige5 rk3576 sbc

### DISCUSSION THREAD

**Discussion by as-jackson** [geerlingguy/sbc-reviews#54] [General]
Hey 👋🏻

Hope I didn't bother you, but we found that there seems to be no tests for rk3576 SBC in the repository.

Can we add tests for [armsom-sige5](https://docs.armsom.org/armsom-sige5) RK3576 SBC? If it takes your time, we can upload the data to the issue following your previous test tutorial, we have a full range of SBC products, they are [armsom-sige1](https://docs.armsom.org/armsom-sige1) RK3528, [armsom-sige3](https://docs.armsom.org/armsom-sige3) RK3568, [armsom-sige7](https://docs.armsom.org/armsom-sige7) (RK3588), if you are interested, we will send them all to you.

**Comment by geerlingguy** [comment]
@as-jackson - I might not have time to test all of them, but if you would be willing to send the armsom-sige7, I would love to test it — please email me at jeff@jeffgeerling.com and I can work out shipping details, thanks!
